docs/evidence/equipment/weapons.md

```diff
@@ -1685,6 +1738,45 @@ The final hit of Raiden's 5-hit string is 2240 in both videos, meaning Emblem's
 **Significance:**  
 Expected results.
 
+#### Xiphos Buff Duration and CD Reset
+
+**By:** Casdela\#5121  
+**Added:** <Version date="2022-11-09" />  
+
+**Finding:**  
+Xiphos and Evenstar gives buff 64 frames after equipped, all timers are reset when removed or when enter/exit domains (including challenge again, abyss try again), allowing the buff to reapply 64 frames after re-equipping or enter/exit domains regardless of the previous timer.
+  
+**Evidence:**  
+[YouTube](https://youtu.be/2bGZheyHrHg)
+Open and check Kazuha's stats after 56 frames (0.933s) shows that the buff wasn't applied.
+However on 3rd attempt after 64 frames (1.067s) checking Kazuha stats shows that the buff was applied.  
+
+[YouTube](https://youtu.be/j6-HC--0M6I)  
+  
+Using Noblesse as control test, enter/exit domain removes all effects but doesn't reset burst and skill cooldown. However, Xiphos timer reset after removed so it reapplies almost instantly.
+  
+**Significance:**  
+Mostly fluff, just to confirm that the Tulaytullah Weapon Series that Xiphos and Evenstar belongs in have 100% buff up-time regardless of the situations, except for the 1.067s delay at the start.
+This info has no practical application other than to test buff timings.
+
+#### Xiphos Buff Snapshotting
+
+**By:** Casdela\#5121  
+**Added:** <Version date="2022-11-09" />  
+
+**Finding:**  
+Xiphos buff snapshots, and it snapshots every 10s.
+
+**Evidence:**  
+[YouTube](https://youtu.be/bEWKhbj3wX4)
+After 9.667s of when Xiphos first buff starts, the buff doesn't update to match new EM value, meaning Xiphos buff is not dynamic, but is a snapshot instead.
+Elegy buff starts 530f (8.833s) after Xiphos buff first applied, lasts for 746f (12.667s) and ends 612f (11.5s) after Xiphos starts snapshot Elegy bonus. Which means Xiphos can snapshot Elegy twice if it has 10s snapshot interval.  
+The Xiphos buff, that snapshot Elegy's 12.667s EM buff, lasts for at least 18.733s, but no more than 1124f+120f=1244f=20.733s. Which match the expected time frame of 10s snapshot interval.
+  
+**Significance:**  
+Confirms that the Tulaytullah Weapon Series gives buff that re-snapshots every 10s.
+On paper it sounds good, but it's unreliable to snapshot any buffs that lasts less than 10s, because there is absolutely no indication whatsoever of when the buff re-snapshots.
```

Comment:
```suggestion
On paper it sounds good, but it's unreliable to snapshot any buff that lasts less than 10s, because there is absolutely no indication whatsoever of when the buff re-snapshots.
```